scripts/build/set-azp-variables.ps1

```diff
@@ -0,0 +1,8 @@
+# Read variables shared between stages and tasks
```

Comment:
between stages and tasks or between stages and jobs?

a stage is made of jobs; a job is made of tasks (steps) - not clear what this comment is about